sui_programmability/examples/games/sources/raffle.move

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// A basic raffle game that depends on Sui randomness.
+///
+/// Anyone can create a new lottery game with an end time and a cost per ticket. After the end time, anyone can trigger
+/// a function to determine the winner, and the owner of the winning ticket can redeem the entire balance of the game.
+///
+module games::raffle {
```

Comment:
This is a nice example! I think it would also be useful to do a variant that doesn't require tickets--determining the winner automatically sends the funds to the winning address. In this setup, I think we'd have a `LinkedTable<u32,address>` inside the `Game` + provide a separate function for cleaning up after the winner is declared.

Or to keep it simple, we could stuff the participants in a `VecMap` or `VecSet` and clearly document that the lottery size is bounded by the object size limit.

Reply:
Added a variant called `small_raffle` that uses `Table`.